fendermint/testing/graph-test/README.md

```diff
@@ -0,0 +1,20 @@
+# Integration test with The Graph
+
+## Prerequisite 
+0. Install `npm` if you haven't done so already.
+1. Instsall `graph` CLI.
+```bash
+npm install -g @graphprotocol/graph-cli
+```
+
+2. Install `mustache`.
+```bash
+npm install -g mustache
+```
```

Comment:
Can `mustache` be added to the `npm` dependencies and be installed automatically?

Reply:
If not, then it needs to be added [tools installed](https://github.com/consensus-shipyard/ipc/blob/main/.github/actions/install-tools/action.yaml) by to the CI workflow, because currently it [fails](https://github.com/consensus-shipyard/ipc/actions/runs/7567669263/job/20607316110?pr=592) with:

```console
2024-01-18T09:45:34.1720300Z > mustache config/testnet.json subgraph.template.yaml > subgraph.yaml && graph codegen && graph build
2024-01-18T09:45:34.1721694Z 
2024-01-18T09:45:34.1789773Z sh: 1: mustache: not found
2024-01-18T09:45:34.1879691Z [cargo-make][2] ERROR - Error while executing command, exit code: 127
```

Reply:
Fixed. Thanks!